### PR TITLE
Fix DCA status with multiple endpoints

### PR DESCRIPTION
**Pull request by FlorianVeaux**
Now:
image

### CODE REVIEW

File: pkg/status/templates/orchestrator.tmpl [hunk 1]
```diff
@@ -31,8 +31,15 @@ Orchestrator Explorer
...
-  {{- range $key, $value := .OrchestratorEndpoints}}
-  {{$key}} - API Key ending with: {{$value}}
+  {{- range $key, $values := .OrchestratorEndpoints}}
+  {{- if gt (len $values) 1}}
+  {{$key}} - API Keys ending with:
+  {{- range $values }}
+      - {{ . }}
+  {{- end}}
+  {{- else}}
+  {{$key}} - API Key ending with: {{index $values 0}}
```
Comment (ogaca-dd) [inline pkg/status/templates/orchestrator.tmpl:41]: What is the purpose of `index` and `0`?

**Review by ogaca-dd — APPROVED** [review]
LGTM.

File: pkg/status/templates/orchestrator.tmpl (hunk 1 above)
Comment (FlorianVeaux) [reply pkg/status/templates/orchestrator.tmpl:41]: values is an array of all api keys.
If the array contains more than one element we iterate over and add one line per api-key.
If the array contains a single element, we use `index $values 0` to retrieve the first item and display it.

**Review by mfpierre — APPROVED** [review]
LGTM 👍

**Pull request merged**